Composer/packages/electron-server/__tests__/setupTests.js

```diff
@@ -1,4 +1,5 @@
-process.env.DEBUG = 'composer*';
```

Comment:
@tonyanziano I removed this. Was it there for a good reason? Just trying to cut down on the noise in the test output!